tests/cds-rdm/test_full_migration.py

```diff
@@ -315,6 +327,12 @@ def author_with_inspire(record):
     """2294138."""
     dict_rec = record.to_dict()
     assert "contributors" in dict_rec["metadata"]
+    assert (
+        dict_rec["created"]
```

Comment:
I assume this is the record without any creation date available? what are we testing against here?

Reply:
Right, We are testing that it was correctly assigned today as creation date

Reply:
something is not working correctly then, I think. That record has a creation date, both in files and in metadata and in files, it shouldn't end up with today's date

Reply:
I removed form the record the creation date which is the field `916`, since for the first version we are using the records creation date and on transforming if it's not present we use today's date it's picking that one, based on this it's correct. I guess that in this cases we should fall back to the files creation date, I will add this logic and test 👍

Reply:
yes exactly, thanks.
Don't remove 916 from all of the records though.
just to clarify, we need:
1. test with 916 to see if it is assigned
2. test with 916 and multiple versions to check if versions get correct creation dates from the files
3. test with no 916 but having files creation dates
4. test with no dates available (probably it shouldn't happen, shall we let it raise the validation error on load then? - I think it will be the safest)